app/src/main/report/components/ReportsList/ReportListColumns/LatestVersionFilter.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+
+import {ChangeEvent} from "react";
+import {DateRangePicker} from "../../../../shared/components/DatePicker/DateRangePicker";
+import {FilterProps} from "../ReportListTable";
+
+const startDate = new Date("2017-03-01T00:00:00");
+const endDate = new Date;
+
+export interface VersionFilterValue {
+    start: Date,
+    end: Date,
+    versionId: string
+}
+
+export const ReportLatestVersionFilter: React.SFC<FilterProps<VersionFilterValue>> = (props: FilterProps<VersionFilterValue>) => {
+
+    const value = props.filter ? props.filter.value : {
+        start: startDate,
+        end: endDate,
+        versionId: ""
+    };
+
+    const onTextChange = (event: ChangeEvent<HTMLInputElement>) => {
+        props.onChange({
+            end: value.end,
+            start: value.start,
+            versionId: event.target.value
+        })
+    };
+
+    const onStartDateChange = (date: Date) => {
+        props.onChange({
+            end: value.end,
+            start: date,
+            versionId: value.versionId
+        })
+    };
+
+    const onEndDateChange = (date: Date) => {
+        props.onChange({
+            end: date,
+            start: value.start,
+            versionId: value.versionId
+        })
+    };
+
+    return <div>
+        <input type={"text"} className={"form-control mb-1 "}
```

Comment:
I feel like searching by ID might be less common than by date range, so maybe swap the order of these two? I see that this keeps it more in line with other columns, but I still think it might be nicer to have the more user friendly one first.